setup.py

```diff
@@ -1,16 +1,25 @@
 from distutils.core import setup
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+with open(path.join(here, 'README.md')) as f:
+    javarun_long_description = f.read()
 
 setup(
-    name='jrun',
+    name='javarun',
     version='0.1.0-dev',
-    author='Philipp Hanslovsky',
+    author='Philipp Hanslovsky, Curtis Rueden',
     author_email='hanslovskyp@janelia.hhmi.org',
-    description='jrun',
+    description='Launch Java code from Python and the CLI, installation-free.',
+    long_description=javarun_long_description,
+    long_description_content_type='text/markdown',
+    license='Public domain',
     url='https://github.com/scijava/jrun',
-    packages=['jrun'],
+    packages=['javarun'],
     entry_points={
         'console_scripts': [
-            'jrun=jrun.jrun:jrun_main'
+            'jrun=javarun.jrun:jrun_main'
```

Comment:
Would just be javarun:jrun_main if jrun_main is imported in __init__.
```suggestion
            'jrun=javarun:jrun_main'
```